autonick.lua

```diff
@@ -0,0 +1,57 @@
+-- automatically nickname dwarves
+--[====[
+
+autonick
+========
+Automatically nickname dwarves according to ``data/init/names.txt``.
+
+One nickname per line, or an empty line. Repeat entries are discarded.
```

Comment:
It would be useful to also accept comments (lines starting with a '#' character)